_component_design/Eyebrow.md

```diff
@@ -0,0 +1,430 @@
+---
+title: Eyebrow
+parent: Navigation
+layout: component
+category: Components
+usage: >
+  The eyebrow is the top menu bar across all breakpoints within the application frame of a Helix interface. The eyebrow provides access to platform-level services and aggregate information across a user's accounts at Rackspace.
+preview-image: preview-images/eyebrow.svg
+status: stable
+resource: true
+helix-ui-css: true
+helix-ui-javascript: true
+last-modified: 2017-08-30
+---
+
+<!-- {% include toc.html %} commented out due to the size of the doc -->
+
+
+<section class="static-section" markdown="1">
+
+## Introduction
+
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Use the eyebrow nav bar to provide access to globally available features, such as account settings, billing, tickets, user management and so on.
+- The eyebrow must not be used for navigating to product features like infrastructure management.
+
+### Best practice
+
+- The <a href="https://one.rackspace.com/display/reach/Pilot+Shared+Navigation">Pilot navigation API</a> is highly recommended for use in UIs which will be presented along side customer facing applications, such as MyRackspace, FAWS, or the Rackspace Cloud Control Panel.
```

Comment:
Change this HTML hyperlink to a Markdown hyperlink.

Reply:
ugh, not sure why I did that, some habits are hard to kick. Updated.